public/js/index.js

```diff
@@ -1,5 +1,32 @@
-function displayMessage() {
```

Comment:
I need some help to implement the tests.

How should I inject `products` into the tests?

```
describe('products', function() {
  var products = [
    {
      id: 1,
      title: "Duff",
      picture: "https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcRkaOwLSA8zTMSancM1OKuXlm7nylciCxyyLxsUUomjZ6WrGahScQ",
      price: "1€"
    },
  ];
  it('should build products', function() {
    expect(buildProducts(products)).toBe('FIXME');
  });
});
```

```
Chrome 52.0.2743 (Mac OS X 10.11.6) ERROR
  Uncaught TypeError: Cannot read property 'appendChild' of null
  at public/js/index.js:2
```